src/main/java/hudson/plugins/emailext/ExtendedEmailPublisherDescriptor.java

```diff
@@ -453,6 +454,13 @@ public void setDefaultContentType(String contentType) {
         }
     }
 
+    public FormValidation doCheckDefaultSuffix(@QueryParameter String value) {
+        if (value.matches("@[A-Za-z0-9.\\-]+") || Util.fixEmptyAndTrim(value)==null)
```

Comment:
Are underscores not allowed in the suffix?

Reply:
according to RFC 1035 (...have as interior characters only letters, digits, and hyphen), underscores are not allowed.

This method was copied from mailer-plugin, so if it should be allowed, it probably should be done in both here and in mailer.

Reply:
Thank you for the clarification.  The PR looks great to me!